indico/modules/events/abstracts/templates/emails/invited_abstract.txt

```diff
@@ -0,0 +1,20 @@
+{% extends 'emails/base.txt' %}
```

Comment:
This template seems to be unused now that you're using the notification template.